.circleci/config.yml

```diff
@@ -7,12 +7,14 @@
 version: 2
 jobs:
   build:
-    machine: true
+    machine:
+      # pin image to older version as "ubuntu-2204:2023.10.1" breaks on "nvm" step
+      image: ubuntu-2204:2023.04.2
     environment:
       DEPLOY_PACKAGES: 1
       DEB: bionic focal
       RPM: el7 el8
-      ST2_VERSION: "3.6dev"
+      ST2_VERSION: "3.9dev"
```

Comment:
Should this be `v3.8.1` or is it meant to be the next st2 version?

Reply:
Updated in https://github.com/StackStorm/st2web/pull/1024/commits/b158679e6c9176e0c72b54933e7c9f1c6aaa9184